src/containers/viewArticle/__test__/ViewArticle.test.js

```diff
@@ -0,0 +1,25 @@
+// React Libraries
+import React from 'react';
+import { Provider } from 'react-redux';
+
+// third-party libraries
+import { shallow, configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+// modules importation
+import ViewArticle from '../ViewArticle';
+import configureStore from '../../../store/configureStore';
+
+configure({ adapter: new Adapter() });
```

Comment:
You don't need to configure enzyme here again, it has been configured in test-setup.js

Reply:
Please ... I don't get you here.